capstone-frontend/src/components/UserSearchList.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Container, Row, Spinner } from 'react-bootstrap';
+import { UserCard } from './UserCard';
+
+export class UserSearchList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      loading: true,
+    }
+  }
+
+  getUsers = async () => {
+    // Get users by searchQuery
+    await fetch(`/api/userSearch?searchTerm=${this.props.searchQuery}`)
+      .then(response => response.json())
+      .then(users => this._isMounted && this.setState({ users, loading: false }))
+      .catch(function (err) {
+        alert(err)
+      });
+  }
+
+  componentDidMount() {
+    this._isMounted = true;
+    this.setState({ loading: true });
+    this.getUsers();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.searchQuery !== prevProps.searchQuery) {
+      this.setState({ loading: true });
+      this.getUsers();
+    }
+  }
+
+  componentWillUnmount() {
+    this._isMounted = false;
+  }
+
+  render() {
+    return (this.state.loading
+      ?
+      (<div className='text-center mt-4'>
+        <Spinner animation='border' role='status' />
+        <br />
+        <h1>Loading...</h1>
```

Comment:
Think it looks cleaner with "Loading..." and just the spinner. But up to you.

Reply:
Since the rest of the browse pages use this spinner formatting, I've decided to leave it as is to remain consistent, but I think it would be a good idea to do a blanket replacement of the "loading" text in another PR.

Reply:
Sounds good!